suites/upgrade/infernalis-x/stress-split/5-workload/readwrite.yaml

```diff
@@ -0,0 +1,15 @@
+meta:
+- desc: |
+   add test details here
```

Comment:
randomized correctness test for rados operations on a replicated pool, using only reads, writes, and deletes